art/art.py

```diff
@@ -552,24 +554,29 @@ def get_font_dic(font_name):
 
     :param  font_name: font's name
     :type font_name:str
-    :return font's dictionary
+    :return: font's dictionary
     """
     return FONT_MAP[font_name][0]
 
 
-def decor(decoration, reverse=False):
+def decor(decoration, reverse=False, both=False):
     """
     Return given decoration part.
 
     :param  decoration: decoration's name
     :type decoration:str
     :param reverse: true if second tail of decoration wanted
     :type reverse:bool
-    :return decor's tail
+    :param both: both tails returning flag
+    :type bool: bool
+    :return: decor's tail or tails as str or list
```

Comment:
anyway now I think it might be confusing.
what about `decor's tail as str or tails as list of str`?

Reply:
Solved in https://github.com/sepandhaghighi/art/pull/138/commits/4b8a4f4d138ed50bc150285db1351fffd3fe9abd